wiki/Tournaments/OFFC/cirno/en.md

```diff
@@ -0,0 +1,225 @@
+| Streamer | ::{ flag=CN }:: [[GB]ChickenGold](https://osu.ppy.sh/users/16586663), ::{ flag=CN }:: [Dr_Shindo](https://osu.ppy.sh/users/30895664), ::{ flag=CN }:: [kitakore](https://osu.ppy.sh/users/33887163), ::{ flag=CN }:: [Naughty_chas](https://osu.ppy.sh/users/15349720), ::{ flag=HK }:: [[SHK] Arcadia](https://osu.ppy.sh/users/13888511) |
+| Commentator | ::{ flag=CN }:: [_eilphy](https://osu.ppy.sh/users/15175276), ::{ flag=CN }:: [5atori Komeiji](https://osu.ppy.sh/users/34282059), ::{ flag=CN }:: [Akari Date](https://osu.ppy.sh/users/32701801), ::{ flag=CN }:: [Amrzs0971](https://osu.ppy.sh/users/30973609), ::{ flag=CN }:: [Dr_Shindo](https://osu.ppy.sh/users/30895664), ::{ flag=CN }:: [eric1388](https://osu.ppy.sh/users/20384257), ::{ flag=CN }:: [FroZZZZZen](https://osu.ppy.sh/users/29235972), ::{ flag=CN }:: [kitakore](https://osu.ppy.sh/users/33887163), ::{ flag=HK }:: [lelxdme](https://osu.ppy.sh/users/13795588), ::{ flag=CN }:: [My Angel Kuroe](https://osu.ppy.sh/users/13217389), ::{ flag=CN }:: [Naughty_chas](https://osu.ppy.sh/users/15349720), ::{ flag=CN }:: [Nosenso](https://osu.ppy.sh/users/30329943), ::{ flag=HK }:: [[SHK] Arcadia](https://osu.ppy.sh/users/13888511), ::{ flag=HK }:: [stllok](https://osu.ppy.sh/users/14817468) |
+| Showcase replay provider | ::{ flag=CN }:: [Ledeau_Fox](https://osu.ppy.sh/users/15816872) |
+
+## Links
+
+- [OFFC official website](https://offfc.github.io)
+- [Match reference book (Tencent Docs)](https://docs.qq.com/doc/DV2VOWlZJdFZWZUpB)
+- [Tournament discussion](https://osu.ppy.sh/community/forums/topics/1961114)
+- [Discord server](https://discord.gg/YYprGehNxD)
+- [Matchup table](https://docs.google.com/spreadsheets/d/14ZFqYajLDRjr863hDo5n5xvGQMoDZPmNwJycgQXr3YQ/edit?gid=868349196#gid=868349196)
+- **[OFFC: cirno Main spreadsheet (Google Docs)](https://docs.google.com/spreadsheets/d/14ZFqYajLDRjr863hDo5n5xvGQMoDZPmNwJycgQXr3YQ)**
+- **[Tournament match information](https://docs.google.com/spreadsheets/d/14ZFqYajLDRjr863hDo5n5xvGQMoDZPmNwJycgQXr3YQ/edit?gid=1807200006)**
+
+## Participants
+
+| Team # | Name | Members |
+| :-: | :-: | :-- |
+| **#1** | 沟槽的烤翅还在开战舰追我 | [**Dura**](https://osu.ppy.sh/users/5185609), [ayaponzu](https://osu.ppy.sh/users/13291701), [suzuka1](https://osu.ppy.sh/users/29785618), [bad kitten](https://osu.ppy.sh/users/13875116), [NaimuTongzi](https://osu.ppy.sh/users/8883892), [chanliuko](https://osu.ppy.sh/users/32289237), [Fyu_Neru](https://osu.ppy.sh/users/6699436), [MyAngelMizuki_](https://osu.ppy.sh/users/15154497) |
```

Comment:
I think these should have flags near usernames. Looking at a few, some members are from different countries.

See https://osu.ppy.sh/wiki/en/Tournaments/IDC/2020#participants for example, still done even if everyone is in the same country.